core/src/main/java/hudson/Util.java

```diff
@@ -193,7 +193,7 @@ public static String loadFile(@Nonnull File logfile, @Nonnull Charset charset) t
 
         StringBuilder str = new StringBuilder((int)logfile.length());
 
-        try (BufferedReader r = new BufferedReader(new InputStreamReader(Files.newInputStream(logfile.toPath()), charset))) {
+        try (BufferedReader r = Files.newBufferedReader(logfile.toPath(), charset)) {
```

Comment:
would be nice to use `Util.fileToPath` to avoid Runtime exceptions. But it can be done in a separate PR

Reply:
I created PR #3219 to use fileToPath().